src/validate.js

```diff
@@ -0,0 +1,27 @@
+const validate = (pathname, toCase) => {
+  const errors = [];
+  const validCases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+
+  if (!pathname.length) {
+    errors.push({
+      message: 'Text to convert is required.'
+      + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  }
+
+  if (!toCase) {
+    errors.push({
+      message: '"toCase" query param is required.'
+      + ' Correct request is: "/<TEXT_TO_CONVERT>?toCase=<CASE_NAME>".',
+    });
+  } else if (!(validCases.includes(toCase))) {
+    errors.push({
+      message: 'This case is not supported.'
+      + ' Available cases: SNAKE, KEBAB, CAMEL, PASCAL, UPPER.',
+    });
+  }
+
+  return errors;
```

Comment:
As for me - i would move validCases and error messages to different file . So if our valid cases will change we won't need to rewrite the logic files.

Reply:
I would also create an object for possible errors